pkg/workload/workload.go

```diff
@@ -180,6 +202,23 @@ func (i *Info) FlavorResourceUsage() map[kueue.ResourceFlavorReference]Requests
 	return total
 }
 
+func filterResources(resources []PodSetResources, excludedPrefixes []string) {
```

Comment:
```suggestion
func dropExcludedResources(resources []PodSetResources, excludedPrefixes []string) {
```
to better reflect that this is a mutating operation